dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/EndpointResource.java

```diff
@@ -46,7 +46,8 @@
             @ApiResponse(code = 422, message = "Endpoint not valid")
     })
     @Override
-    public Response createEndpoint(@ApiParam(hidden = true) @Auth OrganizationPrincipal organizationPrincipal, @Valid @Profiled(profile = EndpointProfile.PROFILE_URI) Endpoint endpoint) {
+    public Response createEndpoint(@ApiParam(hidden=true) @Auth OrganizationPrincipal organizationPrincipal,
```

Comment:
```suggestion
    public Response createEndpoint(@ApiParam(hidden = true) @Auth OrganizationPrincipal organizationPrincipal,
```
For readability, we've tried to be consistent with annotation format. Can we keep spaces around the `=` in all the changes in this PR?

Reply:
add space around = for readability